manager/manager.go

```diff
@@ -37,105 +37,167 @@ import (
 )
 
 type Manager struct {
-	nornsDispatch    map[string]nornDispatcher
-	dispatcherConfig dispatch.DispatcherConfig
+	idFilter         map[string]filterDispatcher
+	urlDispatcher    map[string]filterDispatcher
```

Comment:
```suggestion
	idFilter         map[string]nornFilter
	urlDispatcher    map[string]dispatch.D
```
The dispatcher is not associated with a norn or a filter.  It is only associated with the endpoint it is dispatching to.